README.md

```diff
@@ -30,13 +30,13 @@ transformers:
 * Run `java -jar deobfuscator.jar`
 * Re-run the detection if the JAR was not fully deobfuscated - it's possible to layer obfuscations
 
-Take a look at [USAGE.md](USAGE.md) for more information.
+Take a look at [USAGE.md](USAGE.md) or [wiki](./wiki) for more information.
```

Comment:
I am assuming this is supposed to link the wiki (https://github.com/java-deobfuscator/deobfuscator/wiki); however, this would instead link to https://github.com/java-deobfuscator/deobfuscator/blob/<branch name>/wiki -- probably not intended.

Reply:
Yes, it is supposed to link actual wiki page, btw even if i declare the link relatively, forked repositories won't have wiki, so i have to link directly to the main one

Reply:
Even if you were to create a relative link, I don't think the fork issue would have any noticeable impact. If the person forked the repo with the intention to create a PR, then there shouldn't be a problem. If someone were to create a separately maintained fork, then they'd have to manually change the absolute URL anyway if they wanted to make a separate wiki.

The problem here is that if I were to merge the PR as it is now, that hyperlink would send users to https://github.com/java-deobfuscator/deobfuscator/blob/master/wiki which would give visitors a 404 page.